src/content/docs/cheerp/guides/clientlib.md

```diff
@@ -0,0 +1,327 @@
+---
+title: Clientlib migration guide
+---
+
+Migrating your code can seem like a daunting task. This page aims to list
+common errors you might encounter, and contains a short description of the
+changes and features you should be aware of to make the migration easy and
+painless. This is not a complete list, but it should provide plenty of
+information to get you started, and hopefully provide some guidance if you get
+stuck.
```

Comment:
Not sure how useful this paragraph really is. "Migrating your code can seem like a daunting task" is useless. I don't like describing what pages "aim" to do, that should be self-explanatory based on the table of contents on the side and the title/subtitle of the page!

More constructively:
- Perhaps write "list common errors you might encounter, and contains a short description of the changes and features you should be aware of to make the migration easy and painless" in a more succinct manner and then move it to the `description` of the page?
- "common errors you might encounter" can be a level 2 heading that wraps the common error sections (which should be level 3 headings) on this page. Perhaps "Troubleshooting"?